src/main/scala/group/holmes/analytics/actors/Core.scala

```diff
@@ -0,0 +1,27 @@
+package group.holmes.analytics.actors
```

Comment:
same comment on the name scheme. open to debate on what style you believe is best. After this is settled, we should stick to one across the project.